packages/vscode-extension/src/commonlib/appStudioLogin.ts

```diff
@@ -17,7 +17,7 @@ import { login, LoginStatus } from "./common/login";
 import * as StringResources from "../resources/Strings.json";
 import * as util from "util";
 
-const accountName = "appStudio";
+const accountName = "m365";
```

Comment:
This will affect cache file name.
For now, there are two login types for M365: appStudio, graph. How about graph login telemetry?

Reply:
Got it. Then I actually need 3 values for account type.

Reply:
Good catch! @chagong - I suggest to not change any used variable for now. How about using new variable for telemetry only and adding `TODO` so we can address more post public preview?